### PR TITLE
Added instructions

### PR DESCRIPTION
**Pull request by KlausMikhaelson**
Added instructions in create-tags.md

## ✔️ Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
image

<a href="https://gitpod.io/#https://github.com/Pradumnasaraf/open-source-with-pradumna/pull/186"></a>

### CODE REVIEW

File: pages/How-to/guide/create-tags.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+Navigate to the repository and click on tags <br>
+<img width="341" alt="image" src="https://user-images.githubusercontent.com/100528412/193890608-85e902d5-5f50-49a6-8ad5-4f3370530a25.png">
+
+### Step 3 
+
+Click on `Create a new release` <br> <img width="914" alt="image" src="https://user-images.githubusercontent.com/100528412/193890768-fb29e090-5d07-48bb-92b9-0f860f75d490.png">
+
+### Step 4
+
+Click on `Choose a tag` to choose a tag and `target:` to choose the branch you are targetting.
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/create-tags.md:21]: I think here we need a screenshot to show how it looks.

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

File: pages/How-to/guide/create-tags.md (hunk 1 above)
Comment (KlausMikhaelson) [reply pages/How-to/guide/create-tags.md:21]: Okay will make those changes in a min !

**Comment by KlausMikhaelson** [comment]
Hey @Pradumnasaraf have made the changes as per the instruction

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**